src/backend/marsha/core/tests/test_xapi.py

```diff
@@ -161,16 +148,32 @@ def test_xapi_statement_enrich_statement(self):
                 "definition": {
                     "type": "https://w3id.org/xapi/video/activity-type/video",
                     "name": {"en-US": "test video xapi"},
-                    "extensions": {
-                        "https://w3id.org/xapi/acrossx/extensions/school": "ufr",
-                        "http://adlnet.gov/expapi/activities/course": "mathematics",
-                        "http://adlnet.gov/expapi/activities/module": "001",
-                    },
```

Comment:
Maybe a test showing where and when this data are send would be clearer to me ?

Reply:
They are not send at all any more. We don't split the `context_id` in `school_name`, `course_name` and `course_run` like we did before and we directly send the `context_id` in a `contextActivities.parent` property. See the example in the description.

The `http://adlnet.gov/expapi/activities/course` activity replace the object definition.

Reply:
oh, ok. I see

Reply:
And everything related to the `course` that was stored in the JWT token is deleted because we don't use it any more.